04_numeronym/amitkrishna/numeronym.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	var slice []string
```

Comment:
This is fine, but if you really care about runtime you could optimise by preallocating the size of `numeronyms`:
```go
s := make([]string, len(vals))
for i, val := range vals {
		s[i] =  numeronym(val)
}
```

Reply:
Allocated the size.